### ISSUE TITLE
Can we name this ReplaceFunction or ReplaceFn instead of using Replace

### ISSUE THREAD

**Issue by xiaodaigh** [jwscook/Replace.jl#1]
Replace can be for many things so ReplaceFn would be clearer.

**Comment by jwscook** [comment]
Yes. I'm not the biggest fan of abbreviations like `Fn`. Is `Fn` common in `Core` or `Base`? `ReplaceFunctions.jl` wouldn't be too long, but then `@replacefunction` is too long imo.

I quite like the idea of changing it to `Cuckoo` or something like that because it's memorable if not discoverable and `@cuckoo` is short. Any thoughts?

**Comment by xiaodaigh** [comment]
> Cuckoo

That would work.

**Comment by jwscook** [comment]
Righto. I'll change things over, but before you get too excited I should point you towards the broken tests in the `dev` branch. I can't make it work for all cases - so it might be of no use for you. Falling back to good old multiple dispatch might be the way forward for you.
